### PR TITLE
Query the state of the pull to refresh view

### PR DESCRIPTION
**Pull request by MadsBogeskov**
This is useful when you need to differentiate between full screen loading views and pull to refresh without needing to include local state.

### CODE REVIEW

File: PullToRefresh/PullToRefresh.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ public class PullToRefresh: NSObject {
     
     // MARK: - State
     
-    var state: State = .Initial {
+    public private(set) var state: State = .Initial {
```
Comment (serejahh) [inline PullToRefresh/PullToRefresh.swift:50]: Why did you change it from `internal` to `private`?

**Comment by serejahh** [comment]
Hi @MadsBogeskov! Thanks for the PR. Why did you change it from `internal` to `private`?

File: PullToRefresh/PullToRefresh.swift (hunk 1 above)
Comment (MadsBogeskov) [reply PullToRefresh/PullToRefresh.swift:50]: No particular reason. It might be excessive, however when it still compiled (so it wasn't used outside this class), and it wasn't set to public I assumed it should be considered private.

**Comment by MadsBogeskov** [comment]
No particular reason. It might be excessive, however when it still compiled (so it wasn't used outside this class), and it wasn't set to public I assumed it should be considered private.

**Comment by MadsBogeskov** [comment]
And thank you for the Pod :)

File: PullToRefresh/PullToRefresh.swift (hunk 1 above)
Comment (serejahh) [reply PullToRefresh/PullToRefresh.swift:50]: In our component it's not used, but someone could create a subclass and override the property

Comment (MadsBogeskov) [reply PullToRefresh/PullToRefresh.swift:50]: Now I might be wrong, however, if it is then set to `internal`, and I use it in a Swift module, then even if I subclass it I won't be able to touch it. If it should be possible to subclass and change the state then perhaps it should just be `public`, since there is no `protected` modifier in Swift (afaik)

Comment (serejahh) [reply PullToRefresh/PullToRefresh.swift:50]: That's my fault, I thought `internal` is the same as `protected`
